content/posts/github-issues-as-a-hugo-frontend/index.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Using Github Issues as a Hugo frontend with Github Actions and Netlify"
+date = "2020-12-01"
++++
+
+I got into the habit of dumping quick blog post ideas into issues on my blog's repo. It's a convenient place to iterate on them and share with friends for feedback before actually publishing on my blog post.
+
+![image](100761218-a6cb2280-33c0-11eb-92df-1b52d91cc16e.png)
+
+The drafts keep accumulating, how do I trick myself into publishing more? Perhaps by reducing the effort required for the next step? Let's do it!
+
+## Architecture
+
+My blog is statically generated using [Hugo](https://github.com/gohugoio/hugo), the [code is hosted on Github](https://github.com/shazow/shazow.net), then when a pull request comes in it is built, previewed, and published on merge by [Netlify](https://netlify.com/).
+
+The blog post drafts are posted as Github issues, so there is a clear gap: How do we convert issues into pull requests for Netlify? Enter Github Actions!
+
+## Github Action: Issue to Pull Request
+
+My [full workflow lives here](https://github.com/shazow/shazow.net/blob/master/.github/workflows/publish.yml) if we want to jump ahead, but let's break down the broad strokes.
+
+I decided to trigger the publishing process once an issue is labelled with 'publish', so let's start with that:

```

Comment:
The colon here and below make it look like something is missing. Did you mean to include an image here?

Reply:
OHNO all of my code snippets disappeared, rude!